ibis/backends/pyspark/tests/conftest.py

```diff
@@ -360,6 +325,7 @@ def con(data_dir, tmp_path_factory, worker_id):
 @pytest.fixture(scope="session")
 def con_streaming(data_dir, tmp_path_factory, worker_id):
     backend_test = TestConfForStreaming.load_data(data_dir, tmp_path_factory, worker_id)
+    backend_test._load_data()
```

Comment:
Not 100% sure why this is needed, let me investigate a bit what's going on.

Reply:
thanks!

Reply:
I figured out what the problem is! The pyspark backend loads statefully and the streaming conf ended up reusing the same temp directory from the batch conf, which already exists when the batch conf tried to load data, so the streaming conf skips data loading. Because `tmpdir` is passed as a fixture, I added a line of code that changes the directory naming for the streaming conf.